Khan/math-input
# Don't mangle i18n

## Summary:
In order for build_mobile_perseus.py to extract strings all of the translation functions need to start with 'i18n.'.  If we rename this boject it won't be able to extract those strings.  This PR fixes this by excluding 'i18n' from mangle.  The terser config is almost identical to the one in webapp.

I've also replace the circle-ci build to run `yarn test` with a GitHub Actions check that does the same.

Issue: FEI-3478

## Test plan:
- yarn build
- inspect build/math-input.js, see that 'i18n._()' calls abound
- yarn link
- cd ../webapp/services/static
- yarn link "@khanacademy/math-input"
- ./content_render/tools/build_mobile_perseus.py 
- see that it completes and that khan/mobile had an updated perseus bundle and template files
- rebuild and run the iOS app in Xcode, see that exercises load and work correctly
